plotview.h

```diff
@@ -79,7 +79,13 @@ public slots:
     void addPlot(Plot *plot);
     void emitTimeSelection();
     void extractSymbols(std::shared_ptr<AbstractSampleSource> src);
-    void exportSamples(std::shared_ptr<AbstractSampleSource> src);
+    enum SampleType
```

Comment:
I'm not sure this extra enum is needed so I'd propose https://github.com/miek/inspectrum/commit/b35fcd7fc793aa0f478616b766dba919070dfadc if that looks good to you.